package.json

```diff
@@ -10,34 +10,32 @@
     },
     "scripts": {
         "analyze": "lit-analyzer \"{packages,tools}/*/src/**/!(*.css).ts\"",
+        "analyze:docs": "cem analyze --globs \"packages/**/*.ts\" --exclude \"**/*.d.ts\" --exclude \"**/stories/**\" --exclude \"**/icons/**\" --exclude \"**/elements/**\" --outdir projects/documentation --litelement",
+        "analyze:storybook": "wireit",
         "build": "wireit",
-        "build:clear-cache": "rimraf packages/*/tsconfig.tsbuildinfo && rimraf tools/*/tsconfig.tsbuildinfo",
-        "build:component-inventory": "node ./tasks/build-component-inventory.js",
```

Comment:
The component inventory was really out-of-date and leveraging assets we no longer have in our node_modules resources so if we want to build something similar in the future, we'll need a different way to go about it.